docs/testing-plan.md

```diff
@@ -0,0 +1,130 @@
+# Goal of testing plan
+Manual testing is expensive and we want to create a testing plan to focus on critical parts of the software while automated tests are being developed. 
+
+Some manual tests may be expensive to automate but we should aim to limit this list as much as possible and create as much test coverage to be confident in our deploys.
+
+When testing manual testing we need to keep in mind still both unit tests and integration tests.
```

Comment:
This line seems a bit repetitive. What about something like:
```
When testing manually, we need to keep in mind both unit and integration tests.
```

Reply:
I also don't think we need to get into explaining what unit tests and e2e tests are. If we want this document to be helpful we should keep it as short as possible in my opinion.